mypyc/emitclass.py

```diff
@@ -464,7 +464,8 @@ def generate_new_for_class(cl: ClassIR,
 
 def generate_traverse_for_class(cl: ClassIR,
                                 func_name: str,
-                                emitter: Emitter) -> None:
+                                emitter: Emitter,
+                                has_dict: bool) -> None:
```

Comment:
Would it make sense to store `has_dict` in `ClassIR`?

Reply:
Yeah.